.github/workflows/release-drafter.yml

```diff
@@ -13,8 +13,24 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Setup Yarn
+        run: |
+          npm uninstall -g yarn
+          npm i -g yarn@1.22.10
+          yarn install --frozen-lockfile
+      - name: Generate build
+        run: |
+          yarn build
+          yarn pack --filename oui.tar.gz
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
           draft: true
-          generate_release_notes: true
\ No newline at end of file
+          generate_release_notes: true
+          files: |
+            oui.tar.gz
```

Comment:
The current pipeline unzipping whatever artifacts are inside a tarball. You might want to zip this inside one more tarball.

Reply:
For clarification, you're saying that `oui.tar.gz` should be wrapped inside of another tarball?

Reply:
Yes

Reply:
Done